scanpy/preprocessing/docs.py

```diff
@@ -0,0 +1,88 @@
+doc_norm_bulk = """\
+Normalize total counts per cell.
+
+Normalize each cell by total counts over genes, so that every cell has
+the same total count after normalization.
+
+Similar functions are used, for example, by Seurat [Satija15]_, Cell Ranger
+[Zheng17]_ or SPRING [Weinreb17]_.
+
+Parameters
+----------
+data : :class:`~anndata.AnnData`
+    The annotated data matrix of shape `n_obs` × `n_vars`. Rows correspond
+    to cells and columns to genes.
+cell_sum_after : `float` or `None`, optional (default: `None`)
+    If `None`, after normalization, each cell has a total count equal
+    to the median of the *counts_per_cell* before normalization.
```

Comment:
Here, the quantile case merits more description, right? In the quantile case, the passed quantile will sum up to this `sum_after` constant.

Shall we call it `target_sum` instead? I guess it's the better name but people will be more confused to see that name... But better names should be preferred. So, if you agree, I'd go for `target_sum`.